native/src/components/RedirectContainer.tsx

```diff
@@ -30,8 +31,11 @@ const RedirectContainer = ({ route, navigation }: RedirectContainerProps): React
 
   useEffect(() => {
     // To support potentially older devices taking longer we setup a separate interval to retry the navigation
-    const interval = setInterval(() => {
-      navigateToDeepLink(url)
+    const interval = setInterval(async () => {
+      const noInAppBrowserOpenYet = !(await InAppBrowser.isAvailable())
+      if (noInAppBrowserOpenYet) {
+        navigateToDeepLink(url)
+      }
```

Comment:
:wrench: I find the current naming hard to understand, perhaps the following (a positive wording) would be better?
```suggestion
      const urlRedirected = await InAppBrowser.isAvailable()
      if (!urlRedirected) {
        navigateToDeepLink(url)
      }
```

Reply:
:question: why does `InAppBrowser.isAvailable()` indicate whether it is already open? According to the docs:
> Detect if the device supports this plugin.

AFAICS, it only tells whether we are able to open a link in general. What am I missing?

Reply:
You're right, I misunderstood what it did! The code that I wrote is completely useless, it just doesn't run the interval on devices that can open the `inAppBrowser`, as in it never does. Good catch, thanks!